Use QTensor for bias

dskhudia opened the pull request:
Summary: Use QTesnor for bias tensor

Differential Revision: D15524980

jerryzh168 reviewed (approved):
lgtm

jerryzh168 commented on test/test_quantized.py:
```diff
@@ -469,9 +469,11 @@ def test_qconv(self):
+        b = X_scale * W_scale * (b_init - 0).to(dtype=torch.float)
+
+        X_q = torch.quantize_linear(X, scale=X_scale, zero_point=X_zero_point, dtype=torch.quint8)
+        W_q = torch.quantize_linear(W, scale=W_scale, zero_point=W_zero_point, dtype=torch.quint8)
```
are we going to use qint8 for weight?

dskhudia replied:
Yes. Let me fix it.

dskhudia committed:
Use QTensor for bias (#21038)

Summary:
Use QTesnor for bias tensor
Pull Request resolved: https://github.com/pytorch/pytorch/pull/21038

Differential Revision: D15524980

Pulled By: dskhudia

fbshipit-source-id: ff476919e40fd9ede7fd9fe96648cc69392f43b6
